packages/react/src/checkbox/icons-svg/IconChecked.tsx

```diff
@@ -0,0 +1,19 @@
+import * as styles from "../Checkbox.css";
+export const IconChecked = (
```

Comment:
Components should be Pascal Case and everything else should be camelCase

so let's refactor this into a component

```suggestion
export const IconChecked = () => 
```